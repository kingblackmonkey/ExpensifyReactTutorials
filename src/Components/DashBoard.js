import React from 'react';
import WrapperListExpense  from './ListExpense'
import WrapperExpenseFilters from './ExpenseFIlters'


const DashBoard = ()=>{
    

    return(
   
        <div>
             <p>This Is Your Expenses</p>
             <WrapperExpenseFilters />
             <WrapperListExpense  />



        </div>
      
    )
  }
  
  export default DashBoard;