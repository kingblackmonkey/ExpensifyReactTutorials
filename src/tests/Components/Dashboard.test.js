import React from 'react';
import { shallow} from 'enzyme';
import DashBoard from '../../Components/DashBoard'

test (' should render Dashboard component', ()=>{
    const wrapper = shallow(<DashBoard />)
    expect (wrapper).toMatchSnapshot()
})