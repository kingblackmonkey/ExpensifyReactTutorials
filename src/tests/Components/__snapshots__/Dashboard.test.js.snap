// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render Dashboard component 1`] = `
<div>
  <p>
    This Is Your Expenses
  </p>
  <Connect(ExpenseFilters) />
  <Connect(ListExpense) />
</div>
`;
